### ISSUE TITLE
Add an action to join a linked google meet call

### ISSUE THREAD

**Issue by nheath** [deanishe/alfred-gcal#114]
Most of my meetings are virtual, in a google meet call. Can we get an option to join the meeting directly when activating a calendar entry, rather than opening it in google calendar? This seems related to #90 and #92, but avoids your concerns of complexity parsing arbitrary url, since it would use google's structured data.

**Comment by MartinMalinda** [comment]
Yes this would be great! I tried to add it via cmd modifier but couldn't figure it out.
